pkgs/applications/networking/cluster/mesos/default.nix

```
@@ -25,23 +25,23 @@ let
   });
 
 in stdenv.mkDerivation rec {
-  version = "1.4.1";
+  version = "1.10.0";
   pname = "mesos";
 
   enableParallelBuilding = true;
   dontDisableStatic = true;
 
   src = fetchurl {
     url = "mirror://apache/mesos/${version}/${pname}-${version}.tar.gz";
-    sha256 = "1c7l0rim9ija913gpppz2mcms08ywyqhlzbbspqsi7wwfdd7jwsr";
+    sha256 = "188jkvgykisgzcscv0c34x1zlk1sscp9ld2dvx5361grx6hyifgl";
   };
 
   patches = [
     # https://reviews.apache.org/r/36610/
     # TODO: is this still needed?
     ./rb36610.patch
 
-    # see https://github.com/cstrahan/mesos/tree/nixos-${version}
+    # see https://github.com/ludovicc/mesos/tree/nixos-${version}
     ./nixos.patch
   ];
   nativeBuildInputs = [
```

to solve your immediate issue
```suggestion
  nativeBuildInputs = [
    python2.pkgs.protobuf
```